packages/apptive_grid_form/lib/src/apptive_grid_form.dart

```diff
@@ -492,10 +494,18 @@ class ApptiveGridFormDataState extends State<ApptiveGridFormData> {
           ),
         ),
         Text(
-          localization.sendSuccess,
+          _formData?.successTitle ?? localization.sendSuccess,
           textAlign: TextAlign.center,
           style: Theme.of(context).textTheme.headline4,
         ),
+        if (_formData?.successMessage != null) ...[
+          const SizedBox(height: 4),
+          Text(
+            _formData!.successMessage!,
+            textAlign: TextAlign.center,
+            style: Theme.of(context).textTheme.bodyMedium,
+          ),
```

Comment:
How is the web doing this? Is there now a default message? In that case it should obviously be used instead

Reply:
Currently the web doesn't have a default message here